.changeset/tender-brooms-notice.md

```
@@ -0,0 +1,5 @@
+---
+'@mindfiredigital/mdslide-cli': patch
+---
+
+Test changset workflow
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)